core/src/test/scala/unit/kafka/server/ReplicaFetcherThreadTest.scala

```diff
@@ -437,8 +437,8 @@ class ReplicaFetcherThreadTest {
     thread.doWork()
     assertEquals(2, mockNetwork.epochFetchCount)
     assertEquals(1, mockNetwork.fetchCount)
-    assertEquals("OffsetsForLeaderEpochRequest version.",
-      3, mockNetwork.lastUsedOffsetForLeaderEpochVersion)
+    assertTrue("OffsetsForLeaderEpochRequest version.",
+      mockNetwork.lastUsedOffsetForLeaderEpochVersion >= 3)
```

Comment:
Admittedly I'm not sure why we check the version of the last `OffsetForLeaderEpoch` response is 3 here. I decided to widen the check a bit so this won't break for future versions.

Reply:
I think this was originally using `1` in order to ensure that we were using a version which included the epoch in the response. Since then it looks like it has been updated blindly every time we've bumped the protocol. I'm ok leaving this as is, but we could probably also get rid of it.